bokeh/models/ui/ui_element.py

```diff
@@ -43,7 +43,8 @@
 
 @abstract
 class UIElement(Model):
-    """ Base class for user interface elements. """
+    """ Base class for user interface elements.
+    """
```

Comment:
I don't quite know why, but without this line break in the docstring, Sphinx throws an error: `docstring of bokeh.models.ui.ui_element.UIElement:3:Unexpected indentation.`